<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIOption</class>
 <widget class="QWidget" name="GUIOption">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>138</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="in_matlib_path">
       <property name="text">
        <string>Material Library path</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_matlib_path"/>
     </item>
     <item>
      <widget class="QPushButton" name="b_matlib_path">
       <property name="text">
        <string>Select folder</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="in_unit_m">
       <property name="text">
        <string>Meter unit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="c_unit_m">
       <item>
        <property name="text">
         <string>m</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mm</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="in_unit_m2">
       <property name="text">
        <string>Surface unit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="c_unit_m2">
       <item>
        <property name="text">
         <string>m²</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>mm²</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Resources/pyleecan.qrc"/>
  <include location="../../Resources/pyleecan.qrc"/>
 </resources>
 <connections/>
</ui>
